src/pipx/venv.py

```diff
@@ -184,19 +263,130 @@ def get_python_version(self) -> str:
             .strip()
         )
 
+    def pip_search(self, search_term, pip_search_args):
+        cmd = [str(self.python_path), "-m", "pip", "search"]
```

Comment:
Are you not using the `_run_pip` function because it adds the `-q` argument?

Reply:
Actually it was probably originally an accidental omission, but just testing it now, yes the `-q` argument prevents all useful output from pip search for what I do with it later.

Reply:
Also _run_pip (and util.run() ) only return a returncode, it isn't currently possible to get stdout from them even if the `-q` was not appended.

Reply:
I created run_subprocess() in utils to be a nice wrapper around any subprocess.run that needs stdout and/or stderr captured.